config/logging.yaml

```diff
@@ -0,0 +1,46 @@
+# enable logging output inline in rails console
+:console_inline: true
+# colorize log including level token based on severity
+:colorize: true
+# adds an extra line after each log message with information from which file:line and method
+# was this message logged
+:log_trace: false
+#do not log certain paths (example  /notices/get_new)
+:ignored_paths: []
```

Comment:
No, I believe this is so that certain paths can be ignored such as `/assets/` that crowd the logs with unneeded information.